pagination/helpers.go

```diff
@@ -3,16 +3,31 @@ package pagination
 import (
 	"encoding"
 	"encoding/base64"
+	"errors"
+	"github.com/gazebo-web/gz-go/v7/repository"
+	"github.com/gazebo-web/gz-go/v7/repository/firestore"
 	"time"
 )
 
-// PageSizeGetter holds a method to return the amount of pages requested by users when listing items in an API call.
-type PageSizeGetter interface {
+// Pagination determines the methods needed in a request object to perform pagination.
+type Pagination interface {
+	pageSizeGetter
+	pageTokenGetter
+}
+
+// pageSizeGetter holds a method to return the amount of pages requested by users when listing items in an API call.
+type pageSizeGetter interface {
 	// GetPageSize returns the desired page size. It's using int32 in order to match the method signature from the
 	// generated Go stubs that also return an int32 value.
 	GetPageSize() int32
 }
 
+// pageTokenGetter holds a method to the page requested by a user.
+type pageTokenGetter interface {
+	// GetPageToken returns the requested page token. If the user requests a specific page, this value is not empty.
+	GetPageToken() string
```

Comment:
Is this value expected to be in a specific format, say, `base64`, or are we expecting the implementation to handle that conversion and return the raw value? 

This might be something for us to impose on implementations at the interface level and should document here.